addons/Popochiu/Editor/Importers/Aseprite/docks/AnimationPlayerInspectorDock.tscn

```diff
@@ -0,0 +1,230 @@
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+text = "Output folder"
+
+[node name="OutFolderButton" type="Button" parent="margin/VBoxContainer/Options/OutFolder"]
+margin_left = 108.0
+margin_right = 212.0
+margin_bottom = 20.0
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+text = "[empty]"
+clip_text = true
+
+[node name="OutFile" type="HBoxContainer" parent="margin/VBoxContainer/Options"]
+margin_top = 24.0
+margin_right = 212.0
+margin_bottom = 48.0
+hint_tooltip = "Base filename for spritesheet. In case the layer option is used, this works as a prefix to the layer name."
+
+[node name="Label" type="Label" parent="margin/VBoxContainer/Options/OutFile"]
+margin_top = 5.0
+margin_right = 109.0
+margin_bottom = 19.0
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+text = "Output file name"
+
+[node name="OutFileName" type="LineEdit" parent="margin/VBoxContainer/Options/OutFile"]
+margin_left = 113.0
+margin_right = 212.0
+margin_bottom = 24.0
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+
+[node name="VisibleLayers" type="HBoxContainer" parent="margin/VBoxContainer/Options"]
+margin_top = 52.0
+margin_right = 212.0
+margin_bottom = 92.0
+hint_tooltip = "If active, layers not visible in the source file won't be included in the final image."
+
+[node name="Label" type="Label" parent="margin/VBoxContainer/Options/VisibleLayers"]
+margin_top = 13.0
+margin_right = 118.0
+margin_bottom = 27.0
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+text = "Only visible layers"
+
+[node name="VisibleLayersCheckButton" type="CheckButton" parent="margin/VBoxContainer/Options/VisibleLayers"]
+margin_left = 122.0
+margin_right = 212.0
+margin_bottom = 40.0
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+
+[node name="WipeOldAnimations" type="HBoxContainer" parent="margin/VBoxContainer/Options"]
+margin_top = 96.0
+margin_right = 212.0
+margin_bottom = 136.0
+hint_tooltip = "If active, layers not visible in the source file won't be included in the final image."
+
+[node name="Label" type="Label" parent="margin/VBoxContainer/Options/WipeOldAnimations"]
+margin_top = 13.0
+margin_right = 132.0
+margin_bottom = 27.0
+hint_tooltip = "Set this to OFF if you want to add new animations on top of old ones. Anims with same name will be updated."
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+text = "Wipe old animations"
+
+[node name="WipeOldAnimationsCheckButton" type="CheckButton" parent="margin/VBoxContainer/Options/WipeOldAnimations"]
+margin_left = 136.0
+margin_right = 212.0
+margin_bottom = 40.0
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 2.0
+
+[node name="Import" type="Button" parent="margin/VBoxContainer"]
+margin_top = 238.0
+margin_right = 212.0
+margin_bottom = 258.0
+text = "Import"
+
+[node name="Reset" type="Button" parent="margin/VBoxContainer"]
+margin_top = 262.0
+margin_right = 212.0
+margin_bottom = 282.0
+text = "Reset Preferences"
+flat = true
```

Comment:
This is to mark out that it's a "secondary action". It's to give the "import" button more affordance and avoid people hitting the reset button by mistake (I know we have a confirmation dialog but it's annoying).

Do you think it's wrong?

Reply:
I get your point, and I don't think it is wrong. Maybe is just me, but is kind of confusing to notice that "Reset References" is a button I can click.

Reply:
OK, maybe I can find another style for that. I'll take a look.

Reply:
There is no specific "secondary button" style and I personally don't feel confident overriding Godot's theme. In addition, I have a sweet spot for consistency, so I just made this a button to avoid confusion. I think this works.